Framework/Source/Utils/RenderGraphUI.cpp

```diff
@@ -49,6 +45,7 @@ namespace Falcor
     Gui* gpGui;
```

Comment:
Why do we have some many global variables here? They should be part of the class